cmd/collector/main.go

```diff
@@ -21,12 +21,22 @@ import (
 	"os"
 	_ "time/tzdata"
 
+	"github.com/google/uuid"
 	"github.com/observiq/observiq-otel-collector/collector"
 	"github.com/observiq/observiq-otel-collector/internal/logging"
 	"github.com/observiq/observiq-otel-collector/internal/service"
 	"github.com/observiq/observiq-otel-collector/internal/version"
+	"github.com/observiq/observiq-otel-collector/opamp"
 	"github.com/spf13/pflag"
 	"go.uber.org/zap"
+	"gopkg.in/yaml.v3"
+)
+
+const (
+	ENDPOINT   = "OPAMP_ENDPOINT"
+	AGENT_ID   = "OPAMP_AGENT_ID"
+	SECRET_KEY = "OPAMP_SECRET_KEY"
+	LABELS     = "OPAMP_LABELS"
```

Comment:
these should be lower cased. The linter is yelling at you because they are exported with no comment.